native-image/what-the-dickens/lab/pom.xml

```diff
@@ -70,19 +73,37 @@
                             <buildArgs>
                                 <!-- Provide more memory to Native Image -->
                                 <buildArg>-J-Xmx8G</buildArg>
-
                                 <!-- Replace with line to reduce the startup time of your native executable -->
-
                                 <!-- Replace with line to produce a mostly static native executable -->
-
                             </buildArgs>
-
-                            <!-- Uncomment the next line for Native Image development builds -->
                             <!-- <quickBuild>true</quickBuild> -->
-
-                            <!-- Uncomment the next line for verbose output from Native Image -->
-                            <!-- <verbose>true</verbose> -->
-
+                        </configuration>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+        <profile>
+            <id>mostly-static</id>
```

Comment:
I added a separate profile to test this Docker build with a GitHub workflow pipeline. No changes were required to the text in README. I only updated the Maven build command in native-image/what-the-dickens/lab/02-smaller-containers/Dockerfile  to `RUN ./mvnw --no-transfer-progress -Pnative,mostly-static -DskipTests=true clean package`.